docs/stacks-guide-aws.md

```diff
@@ -158,25 +160,25 @@ provider.aws.crossplane.io/aws-provider created
 
 When configured in AWS, WordPress resources map to an EKS cluster and an RDS
 database. In order to make the RDS instance accessible from the EKS cluster,
-they both need to live within the same VPC. However VPC is not the only AWS
+they both need to live within the same VPC. However, a VPC is not the only AWS
 resource that needs to be created to enable inter-resource connectivity. In
-general, a **Network Configuration**, which is built of a set of VPCs, Subnets,
+general, a **Network Configuration**, which consists of a set of VPCs, Subnets,
 Security Groups, Route Tables, IAM Roles and other resources, is required for
-this purpose. For more information, see [AWS resource connectivity] design
+this purpose. For more information, see [AWS resource connectivity][] design
```

Comment:
empty `[]`, is that intentional?

Reply:
Yeah; it's the only documented syntax for implicit link naming [here](https://daringfireball.net/projects/markdown/syntax#link) and I was feeling conservative